code/modules/cm_marines/equipment/sentries.dm

```diff
@@ -292,6 +297,11 @@
 	var/immobile = 0 //Used for prebuilt ones.
 	var/datum/ammo/bullet/turret/ammo = /datum/ammo/bullet/turret
 	var/obj/item/projectile/in_chamber = null
+	var/alerts_on = TRUE
+	var/last_alert = 0
+	var/alert_delay = 20 //20 seconds
```

Comment:
```suggestion
	#define TURRET_ALERT_DELAY 20 //20 seconds
```

Reply:
alert delay can be a variable because you might want to change it on other types. Also the define shouldnt go here.

Reply:
The define shouldn't go here for sure, but so far this is a constant, not a variable. It's not changed anywhere else.